content/en/docs/concepts/overview/components.md

```diff
@@ -3,6 +3,11 @@ reviewers:
 - lavalamp
 title: Kubernetes Components
 content_type: concept
+description: >
+  A Kubernetes cluster consists of a set of machines called nodes.
+  The nodes host the Pods that represent the application components and run the containerized applications.
```

Comment:
Please edit. This seems too much for a description.
A Kubernetes cluster consists of the components that represent the control plane and [set of machines called ] nodes.

Reply:
Another idea:
Components that comprise a cluster such as node and control plane.